core/conversion/converters/impl/element_wise.cpp

```diff
@@ -144,6 +144,41 @@ auto element_wise_registrations TRTORCH_UNUSED =
                     LOG_DEBUG("Output tensor shape: " << out->getDimensions());
                     return true;
                   }})
+        .pattern({"aten::clamp(Tensor self, Scalar? min=None, Scalar? max=None) -> (Tensor)",
```

Comment:
Could we possibly lower to hardtanh or is the functionality different?

Reply:
hardtanh looks a bit different [https://pytorch.org/docs/stable/generated/torch.nn.Hardtanh.html](url)
hardtanh compares the elements to -1 and 1 and replaces with default values (thresholds) where as clamp compares the input elements to thresholds directly. They both probably can match in certain scenarios

Reply:
hardtanh lets you configure the min and max values. really my point is we should be able to lower clamp and hardtanh to the same clip activation converter. This is something we could leave for later though

Reply:
Ideally our converter library should be as small as possible and we should do as much as we can through lowering